.obs/specfile/elemental-operator.spec

```diff
@@ -59,6 +59,9 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 %package -n elemental-register
 Summary: The registration client
 
+Requires: dmidecode
+Requires: libopenssl1_1
```

Comment:
Libraries must not appear in `Requires`. The build process has tooling in place to automatically find library dependencies.